### PR TITLE
Github action with multi-CPU architecture support and multi docker file support.

### PR DESCRIPTION
**Pull request by skabashnyuk**
- Build and push to docker registry Dockerfile under linux/amd64,linux/s390x,linux/arm64,linux/ppc64le
 - Build and push to docker registry rhel.Dockerfile under linux/amd64
 - Allow to configure with secrets DOCKER_USERNAME, DOCKER_PASSWORD, DOCKER_IMAGE, DOCKER_REPO
 - Same workflow for master and prs
 - Is able to push tags

Results https://quay.io/repository/skabashn/configbump?tab=tags

### CODE REVIEW

**Comment by benoitf** [comment]
could we add `arm64` and `ppc64le` ?

**Comment by skabashnyuk** [comment]
>could we add arm64 and ppc64le ?

I don't see a big problem. 
several thoughts:
 - take more time to build.
 - we need to ensure it's stable. I would say two architectures that repeatedly stable built - it's already a win. 
 - we need to ensure all other images build on that architectures too. However, for this concrete case, it might be useful to build it unconditionally.

**Comment by skabashnyuk** [comment]
CC @nickboldt @vparfonov @ibuziuk

**Comment by benoitf** [comment]
more time to build:
--> it's not done in parallel ?

**Comment by benoitf** [comment]
> we need to ensure all other images build on that architectures too. However, for this concrete case, it might be useful to build it unconditionally.

idea was to bring amd64, arm64 and the two from IBMs 
Usually when it's starting to work with more than once arch it's little thing to adapt.

File: build/dockerfiles/Dockerfile [hunk 1]
```diff
@@ -10,23 +10,71 @@
...
+ARG VCS_REF
+ARG VERSION
+
+LABEL maintainer="Eclispe Che team" \
+  org.opencontainers.image.created=$BUILD_DATE \
+  org.opencontainers.image.url="https://github.com/che-incubator/configbump" \
+  org.opencontainers.image.source="https://github.com/che-incubator/configbump" \
+  org.opencontainers.image.version=$VERSION \
+  org.opencontainers.image.revision=$VCS_REF \
+  org.opencontainers.image.vendor="Eclispe Che team" \
```
Comment (sleshchenko) [inline build/dockerfiles/Dockerfile:60]:
```suggestion
  org.opencontainers.image.vendor="Eclipse Che team" \
```

File: build/dockerfiles/Dockerfile [hunk 2]
```diff
@@ -10,23 +10,71 @@
...
+ARG VERSION=dev
+RUN env $(cat /tmp/.env | xargs) go build -a -ldflags '-w -s' -a -installsuffix cgo -o configbump cmd/configbump/main.go
+
+FROM --platform=${TARGETPLATFORM:-linux/amd64} alpine:3.12
+
+ARG BUILD_DATE
+ARG VCS_REF
+ARG VERSION
+
+LABEL maintainer="Eclispe Che team" \
```
Comment (sleshchenko) [inline build/dockerfiles/Dockerfile:54]:
```suggestion
LABEL maintainer="Eclipse Che team" \
```

**Comment by skabashnyuk** [comment]
@benoitf @sparkoo added ``linux/arm64,linux/ppc64le``. Time of build increased from 12m->15m

**Comment by benoitf** [comment]
@skabashnyuk maybe you could spin `Docker Buildx rhel` with another job so it's done in parallel

**Comment by skabashnyuk** [comment]
>@skabashnyuk maybe you could spin Docker Buildx rhel with another job so it's done in parallel

I'll experiment on that. At this moment I'm afraid errors during concurrent pushes. And logout probably has to be mode on another job too.

**Pull request merged**